docs/user/troubleshooting.asciidoc

```diff
@@ -68,3 +68,32 @@ Open Kibana Logs and search for the HTTP server response for the endpoint. It lo
 }
 ----
 You are interested in the https://www.elastic.co/guide/en/ecs/current/ecs-tracing.html#field-trace-id[trace.id] field, which is a unique identifier of a trace. The `trace.id` provides a way to group multiple events, like transactions, which belong together. You can search for `"trace":{"id":"9b99131a6f66587971ef085ef97dfd07"}` to get all the logs that belong to the same trace. This enables you to see how many {es} requests were triggered during the `9b99131a6f66587971ef085ef97dfd07` trace, what they looked like, what {es} endpoints were hit, and so on.
+
+== Trace {es} query to the origin in {kib}
+Sometimes the {es} server might be slowed down by the execution of an expensive query. Such queries are logged to {es}'s https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules-slowlog.html#search-slow-log[Search Slow Log] file. But there is a problem: it's impossible to say who triggered a slow search request - a {kib} instance or a user accessing an {es} endpoint directly.
```

Comment:
There should be a syntax to link to the ES docs and have the correct version automatically applied instead of defaulting to `current`. I think it's something like this (haven't tested):

```
{ref}/index-modules-slowlog.html[Search Slow Log]
```

Reply:
++, The correct syntax for the [current ES](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc#L1) docs is: 
`{ref}` -> `https://www.elastic.co/guide/en/elasticsearch/reference/{branch}`